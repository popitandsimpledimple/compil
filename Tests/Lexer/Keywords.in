and
Array
aS
aSm
BEGIN
BREAK
CASE
CLASS
CONST
CONSTRUCTOR
CONTINUE
DESTRUCTOR
DIV
DISPOSE
DO
DOWNTO
ELSE
END
EXCEPT
EXIT
EXPORTS
FALSE
FILE
FINALIZATION
FINALLY
FLOAT
FOR
FORWARD
FUNCTION
GOTO
IF
IMPLEMENTATION
IN
INLINE
INHERITED
INITIALIZATION
INTERFACE
IS
LABEL
LIBRARY
MOD
NEW
NOT
OBJECT
OF
ON
OPERATOR
OR
OUT
PACKED
PROCEDURE
PROGRAM
PROPERTY
RAISE
READ
READLN
RECORD
REPEAT
SELF
SET
SHL
SHR
STRING
THEN
THREADVAR
TO
TRUE
TRY
TYPE
UNIT
UNTIL
USES
VAR
WHILE
WITH
WRITE
WRITELN
XOR