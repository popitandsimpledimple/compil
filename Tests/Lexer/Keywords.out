1	1	Keyword	AND	and
2	1	Keyword	ARRAY	Array
3	1	Keyword	AS	aS
4	1	Keyword	ASM	aSm
5	1	Keyword	BEGIN	BEGIN
6	1	Keyword	BREAK	BREAK
7	1	Keyword	CASE	CASE
8	1	Keyword	CLASS	CLASS
9	1	Keyword	CONST	CONST
10	1	Keyword	CONSTRUCTOR	CONSTRUCTOR
11	1	Keyword	CONTINUE	CONTINUE
12	1	Keyword	DESTRUCTOR	DESTRUCTOR
13	1	Keyword	DIV	DIV
14	1	Keyword	DISPOSE	DISPOSE
15	1	Keyword	DO	DO
16	1	Keyword	DOWNTO	DOWNTO
17	1	Keyword	ELSE	ELSE
18	1	Keyword	END	END
19	1	Keyword	EXCEPT	EXCEPT
20	1	Keyword	EXIT	EXIT
21	1	Keyword	EXPORTS	EXPORTS
22	1	Keyword	FALSE	FALSE
23	1	Keyword	FILE	FILE
24	1	Keyword	FINALIZATION	FINALIZATION
25	1	Keyword	FINALLY	FINALLY
26	1	Keyword	FLOAT	FLOAT
27	1	Keyword	FOR	FOR
28	1	Keyword	FORWARD	FORWARD
29	1	Keyword	FUNCTION	FUNCTION
30	1	Keyword	GOTO	GOTO
31	1	Keyword	IF	IF
32	1	Keyword	IMPLEMENTATION	IMPLEMENTATION
33	1	Keyword	IN	IN
34	1	Keyword	INLINE	INLINE
35	1	Keyword	INHERITED	INHERITED
36	1	Keyword	INITIALIZATION	INITIALIZATION
37	1	Keyword	INTERFACE	INTERFACE
38	1	Keyword	IS	IS
39	1	Keyword	LABEL	LABEL
40	1	Keyword	LIBRARY	LIBRARY
41	1	Keyword	MOD	MOD
42	1	Keyword	NEW	NEW
43	1	Keyword	NOT	NOT
44	1	Keyword	OBJECT	OBJECT
45	1	Keyword	OF	OF
46	1	Keyword	ON	ON
47	1	Keyword	OPERATOR	OPERATOR
48	1	Keyword	OR	OR
49	1	Keyword	OUT	OUT
50	1	Keyword	PACKED	PACKED
51	1	Keyword	PROCEDURE	PROCEDURE
52	1	Keyword	PROGRAM	PROGRAM
53	1	Keyword	PROPERTY	PROPERTY
54	1	Keyword	RAISE	RAISE
55	1	Keyword	READ	READ
56	1	Keyword	READLN	READLN
57	1	Keyword	RECORD	RECORD
58	1	Keyword	REPEAT	REPEAT
59	1	Keyword	SELF	SELF
60	1	Keyword	SET	SET
61	1	Keyword	SHL	SHL
62	1	Keyword	SHR	SHR
63	1	Keyword	STRING	STRING
64	1	Keyword	THEN	THEN
65	1	Keyword	THREADVAR	THREADVAR
66	1	Keyword	TO	TO
67	1	Keyword	TRUE	TRUE
68	1	Keyword	TRY	TRY
69	1	Keyword	TYPE	TYPE
70	1	Keyword	UNIT	UNIT
71	1	Keyword	UNTIL	UNTIL
72	1	Keyword	USES	USES
73	1	Keyword	VAR	VAR
74	1	Keyword	WHILE	WHILE
75	1	Keyword	WITH	WITH
76	1	Keyword	WRITE	WRITE
77	1	Keyword	WRITELN	WRITELN
78	1	Keyword	XOR	XOR
